packages/server/src/oauth/token.ts

```diff
@@ -607,8 +607,7 @@ async function sendTokenResponse(res: Response, login: Login, client?: ClientApp
     fhircastProps['hub.topic'] = topic;
   }
 
-  const decodedAccessToken = await verifyJwt(tokens.accessToken);
```

Comment:
This is a JWT that we ourselves just generated, so we do not need to re-verify it.